### DISCUSSION TITLE
Junction Deviation?

### DISCUSSION THREAD

**Discussion by Markalorian** [Jyers/Marlin#1929] [Ideas]
Any plans to enable junction deviation or offer a version with it enabled?

Edit: sorry this post was meant to be in a specific fork for the Voxelab Aquila. Please delete.

**Comment by Tapap84** [comment]
So I'm giving junction deviation a go and it dramatically changed my print quality. I was printing with 1200 acceleration and 12 jerk. Sometimes I'd go 1500 acceleration but jerk was always 10 to 14. I always had a slight buldge on 90° corners but nothing bad at all. With junction deviation on I ended up cranking it up and damn. 1500 acceleration with 0.2 junction deviation and my corners are sharp. It doesn't like cura doing the retraction but 0.2 junction had almost no stringing compared to the 0.08 junction. I switched to firmware retraction and the retraction are so much better.

Acceleration 1500
Junction 0.20
Firmware retraction
3mm (microswiss hotend and cht 0.6mm nozzle)
25mm/s no z hop
![20220410_143803](https://user-images.githubusercontent.com/79718111/162634518-f4397380-d6d6-495c-acce-6f4c5ba05dc8.jpg)
